### PR TITLE
Fix CPINFOEXW definition

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs [hunk 1]
```diff
@@ -19,14 +19,14 @@ private unsafe struct CPINFOEXW
             internal fixed byte LeadByte[12];
             internal char UnicodeDefaultChar;
             internal uint CodePage;
-            internal fixed byte CodePageName[260];
+            internal fixed char CodePageName[260];
```
Comment (jkotas) [inline src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs]: This is the actual bug

File: src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@ internal partial class Interop
     internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, EntryPoint = "GetCPInfoExW")]
-        private extern static unsafe int GetCPInfoExW(uint CodePage, uint dwFlags, CPINFOEXW* lpCPInfoEx);
+        private extern static unsafe Interop.BOOL GetCPInfoExW(uint CodePage, uint dwFlags, CPINFOEXW* lpCPInfoEx);
```
Comment (jkotas) [inline src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs]: Cleanup for consistency with GetCPInfo

**Comment by jkotas** [comment]
Related to https://github.com/dotnet/corefx/issues/37111

File: src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs (hunk 1 above)
Comment (danmosemsft) [reply src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs]: `internal fixed char CodePageName[MAX_PATH];` ?

File: src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj [hunk 3]
```diff
@@ -36,6 +36,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
       <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Interop.BOOL.cs">
+      <Link>Common\CoreLib\Interop\Windows\Interop.BOOL.cs</Link>
+    </Compile>
```
Comment (danmosemsft) [inline src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj]: would need `<Compile Include="$(CommonPath)\CoreLib\Interop\Windows\Kernel32\Interop.MAX_PATH.cs">..`

File: src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs (hunk 1 above)
Comment (stephentoub) [reply src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs]: @jkotas, how did you find this? Is a test failing? Something getting corrupted? Crash?

Comment (jkotas) [reply src/Common/src/Interop/Windows/Kernel32/Interop.GetCPInfoEx.cs]: I wanted to see what the `CodePageName` returned by the OS is. I copied this interop definition from CoreFX to my local test and the string created from `byte CodePageName` was garbage. It is how I have noticed the bug.

**Pull request merged**